README.md

```diff
@@ -10,6 +10,7 @@ The leaderboard website displays scores submitted by different players. It also
 - JavaScipt
 - ES6
 - Webpack
+- API
 
 ## Live Demo
```

Comment:
- [ ] Your Live Demo link is broken. Kindly fix that.
![Screenshot from 2021-10-16 16-15-23](https://user-images.githubusercontent.com/65476715/137592768-cae48b69-1723-423c-9343-2e35879f95df.png)